README.md

```diff
@@ -25,12 +25,14 @@ If you want to contribute read the [contribution guideline](CONTRIBUTING.md) and
 Learn more about the different network by reading the [upgrade documentation](/docs/identity/upgrade.md#philosophy)
 
 ### Provisioning of workloads
-ZOS does not expose an interface, instead of wait for reservation to happen on a trusted
-source, and once this reservation is available, the node will actually apply it to reality. You can start reading about [provisioning](docs/provision) in this document.
+ZOS does not expose an interface, instead it waits for reservation to happen on a trusted source, and once this reservation is available, the node will actually apply it to reality. You can start reading about [provisioning](docs/provision) in this document.
 
 Currently zos supports two different source:
-- BCDB
-- BCDB mock
+#### BCDB
```

Comment:
I feel this section is redundant with the `Grid Networks` section just above. It also talk about pretty low level things link kernel parameters. I don't think this belong on home page readme.